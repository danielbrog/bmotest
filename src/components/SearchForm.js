import React from "react";

const SearchForm = () => (
  <div>
    <div className="content">Search Restaurants</div>
    <input type="search" name="Search Cities" placeholder="City"></input>
  </div>
);

export default SearchForm;
