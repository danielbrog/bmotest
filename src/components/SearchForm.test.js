import React from "react";
import SearchForm from "./SearchForm";
import configureStore from "redux-mock-store";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
import mockStore from "../mocks/mockStore.json";

const configStore = configureStore([]);

test("SearchForm renders", () => {
  const store = configStore(mockStore);
  const testRenderer = renderer
    .create(
      <Provider store={store}>
        <SearchForm />
      </Provider>
    )
    .toJSON();

  expect(testRenderer).toMatchSnapshot();
});
