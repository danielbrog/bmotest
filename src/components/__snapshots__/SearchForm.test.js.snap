// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders 1`] = `
<div
  className="search__items"
  role="form"
>
  <div
    className="search__city"
  >
    <label
      className="search__city__label"
      htmlFor="city"
    >
      City:
       
    </label>
    <input
      className="search__city__input"
      id="city"
      name="Search Cities"
      onChange={[Function]}
      placeholder="City"
      type="search"
    />
    <button
      className="search__city__button"
      onClick={[Function]}
      type="button"
    >
      Search Restaurants
    </button>
  </div>
  <div
    className="search__refine"
  >
    <label
      className="search__refine__label"
      htmlFor="refine"
    >
      Refine:
       
    </label>
    <input
      className="search__refine__input"
      id="refine"
      name="Refine Search"
      onChange={[Function]}
      placeholder="Name / Street / Area"
      type="search"
    />
    <button
      className="search__refine__button"
      onClick={[Function]}
      type="button"
    >
      Refine Search
    </button>
  </div>
</div>
`;
